---
layout: post
title: BEECHGENOME | break through in genome quality
summary: The last weeks brought enormous progress for a high quality beech genome.
featured-img: beech-bw
---

# News
The last weeks brought enormous progress for a high quality beech genome. Asked what a "nice reference genome means" - this is what our bioinformatician Isabelle Lesur had to say: "Assembly size= 542715598bp, N50 size= 16613141bp, number= 13; the largest scaffold is 33Mb long and the 26 largest scaffolds represent 80% of the assembly. Oh Yes, and number of "N": ZERO !!" This new reference has immediately been put to use in the last weeks in the framework of our H2020 project GenTree for SNP calling. 93% of our GenTree reads could be mapped producing a whopping 22 millions SNPs - though unfiltered at this stage.

Within BEECHgenome the new reference will also be put to use in the next months for resequencing *Fagus orientalis* as well as some interesting freak phenotypes from *Fagus sylvatica*. We plan to publish the genome and subsequent studies starting spring 2020.

# Project Background

The project *BEECHgenome - Genomic variation in common beech: analysis of the adaptation and adaptability of a forest species of great ecological and economic importance threatened by climate change* - has been running since 2017 and will continue until 2020. It is funded as part of the France genomics call for large sequencing projects, and led by INRA-URFM by Dr. Ivan Scotti. In BEECHgenome we have three objectives: (1) to establish a reference genomic sequence for the common beech; (2) obtain high-density polymorphism data by pool-sequencing populations obtained across Europe and Asia minor including *Fagus orientalis*; (3) identify patterns of local multi-scale adaptation, from the stand to the distribution area. The BEECHgenome project has close ties with the ongoing H2020 program GenTree, in which  Isabelle Lesure of my team is working as a PostDoc and analyses the beech genomic data of both projects. 

Common beech (*Fagus sylvatica L.*) is an important keystone forest species, representing more than 15% of Europe's forests and of great commercial importance. It is the subject of many high quality research programs in ecology, forest science, genetics and ecophysiology. Despite this, there is a glaring lack of genomic resources and knowledge on the genomic basis of adaptation in this species. Following the 2018 drought in central Europe that significantly damaged many beech stands, it is evident that this resource will be key to assess whether intraspecific genomic variability can in fact mitigate drought effects in the future.

# Participating group members
* Dr. Isabelle Lesur (bioinformatics)
* Dr. Martin Leberecht (sample collection)

# Co-workers
* Dr. Katrin Heer (UMR)
* Dr. Felix Gugerli (WSL)
* Dr. Christian Rellstab (WSL)
* Dr. Benjamin Dauphin (WSL)

# Collaboration partners and Institutions
INRA Avignon
* Dr. Ivan Scotti

INRA Bordeaux
* Prof. Dr. Christophe Plomion